src/v2/Components/BankDebitForm/BankDebitForm.tsx

```diff
@@ -50,10 +52,14 @@ export const BankDebitForm: FC<Props> = ({ order }) => {
       return
     }
 
+    // Disable the "leave/reload site?" confirmation dialog as we're about to
+    // confirm Stripe payment setup which leaves and redirects back.
+    window.removeEventListener("beforeunload", preventHardReload)
```

Comment:
To avoid the confirmation alert from popping up during redirect.

![Screen Shot 2022-05-13 at 3 55 45 PM](https://user-images.githubusercontent.com/796573/168382442-8b85a4ff-edbb-446d-9b91-b023e30cefe4.png)